### ISSUE TITLE
Suggestions for Enhancing Accessibility Clarity in the WSG Draft

### ISSUE THREAD

**Issue by goodwitch** [w3c/sustainableweb-wsg#205]
This issue is being raised by the [Accessible Platform Architectures (APA) WG]( https://www.w3.org/WAI/about/groups/apawg/)

1. **Web Sustainability Guideline** _(in general)_
https://www.w3.org/TR/2025/DNOTE-web-sustainability-guidelines-20251119/
    - **Comment:** WSG is so big, people are encouraged to start by focusing on one area. Because of this, we are concerned that the danger of someone focusing so much on sustainability that they sacrifice accessibility is a little too high. We think it would be valuable for the WSG (on a future draft) to ensure that anyone looking at just one Guideline (that is related to a11y) will know that they should not sacrifice a11y for sustainability.
2. **1.5 Relationships 
Relationship to Web Content Accessibility Guidelines (WCAG)**
https://www.w3.org/TR/2025/DNOTE-web-sustainability-guidelines-20251119/#relationships
    - **Comment** We'd love to see a stronger statement about how to handle conflicts between WSG and WCAG. I was thinking that could happen here in [1.5 ](https://www.w3.org/TR/web-sustainability-guidelines/#relationships)Relationship to Web Content Accessibility Guidelines (WCAG)
        - **Current:** Web Content Accessibility Guidelines (WCAG) provides recommendations for making web content more accessible [[WCAG](https://www.w3.org/TR/web-sustainability-guidelines/#bib-wcag)]. As WSG[ contains accessibility considerations](https://www.w3.org/TR/web-sustainability-guidelines/#accessibility) due to the social impacts of sustainability overlapping with inclusive design/development, the impacts of affected parties have been taken into consideration whenever possible.
        - **Proposed:** Web Content Accessibility Guidelines (WCAG) provides recommendations for making web content more accessible [[WCAG](https://www.w3.org/TR/web-sustainability-guidelines/#bib-wcag)]. WSG complements WCAG by showing where sustainability and accessibility align. If a WSG recommendation conflicts with WCAG, WCAG should take precedence.

3. **A.1 Accessibility**
https://www.w3.org/TR/2025/DNOTE-web-sustainability-guidelines-20251119/#accessibility
**Comment:** Several WSG Success Criteria that we believe are related to a11y are not listed in [A.1](https://w3c.github.io/sustainableweb-wsg/#accessibility) Accessibility. we assume these items are missing because this is an early draft, and that the WSG team will update it in the near future. Examples of missing items:
    - [2.3](https://w3c.github.io/sustainableweb-wsg/#understand-the-impact-for-non-users) Understand the impact for non-users
    - [2.4](https://w3c.github.io/sustainableweb-wsg/#integrate-sustainability-into-every-stage-of-the-ideation-process) Integrate sustainability into every stage of the ideation process
        - Success Criterion: Participation and Testing
            - (In "Additional Information", add "Accessibility" as a benefit, and recommend involving people with disabilities)
    - [2.6](https://w3c.github.io/sustainableweb-wsg/#minimize-non-essential-content-interactivity-or-journeys) Minimize non-essential content, interactivity, or journeys
        - (We'd like to see something added about when sustainability conflicts with accessibility, that accessibility should take precedence.)
        - Success Criterion:  Eliminate the non-essential
            - (We'd like to see a note added about disabilities, because what may be non-essential for one person may be essential for a person with x, y, or z disability).
    - [2.7](https://w3c.github.io/sustainableweb-wsg/#use-decorative-design-with-care) Use decorative design with care
    - [2.9](https://w3c.github.io/sustainableweb-wsg/#design-to-assist-and-not-to-distract) Design to assist and not to distract
    - [2.10](https://w3c.github.io/sustainableweb-wsg/#use-established-design-patterns-and-essential-components) Use established design patterns and essential components
    - There are more missing…

**Comment by jenstrickland** [comment]
As asked by @mgifford  in the SWING meeting on 21 January 2026, I am noting that I reviewed #213 and provided feedback in that pull request.

Additionally, I responded to the concerns expressed on this issue and in the meeting on 15 January 2026, which is documented in the minutes for that day.
